src/screens/SplashScreen/index.js

```diff
@@ -48,9 +49,14 @@ const SplashScreen = () => {
     try {
       const id = await getUserId();
       const anonymousUserId = await getAnonymousUserId();
-      setProfileId({
+      const token = TokenStorage.get(ITokenEnum.token);
+      const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+
+      setAuth({
         anonProfileId: anonymousUserId,
-        profileId: id
+        profileId: id,
+        token,
+        anonymousToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of token and anonymousToken is a good step towards better user authentication. However, make sure that these tokens are validated before being used.
```diff
-      setProfileId({
-         anonProfileId: anonymousUserId,
-        profileId: id
-       });
+      const token = TokenStorage.get(ITokenEnum.token);
+      const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+
+      setAuth({
+         anonProfileId: anonymousUserId,
+         profileId: id,
+         token,
+         anonymousToken
+      });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->